src/app/shared/palette/order-summary/fee-value.model.ts

```diff
@@ -0,0 +1,5 @@
+import { Fee } from './fee.model';
+
+export class FeeValue {
```

Comment:
is this FeeValue needed? 
Can't the field Fees of OrderSummary be a Fee[] instead of a FeeValue[]?

Reply:
Yes it is needed as these are the objects inside a collection (id, value).